File: src/Z.Core/System.Double/System.Double/Double.IsInfinity.cs
// Description: C# Extension Methods Library to enhances the .NET Framework by adding hundreds of new methods. It drastically increases developers productivity and code readability. Support C# and VB.NET
// Website & Documentation: https://github.com/zzzprojects/Z.ExtensionMethods
// Forum: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License: https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: http://www.zzzprojects.com/
// Copyright © ZZZ Projects Inc. 2014 - 2016. All rights reserved.
using System;

public static partial class Extensions
{
    /// <summary>
    ///     Returns a value indicating whether the specified number evaluates to negative or positive infinity.
    /// </summary>
    /// <param name="d">A double-precision floating-point number.</param>
    /// <returns>true if  evaluates to  or ; otherwise, false.</returns>
    public static Boolean IsInfinity(this Double d)
    {
        return Double.IsInfinity(d);
    }
}

File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Core/System.Double/System.Double/Double.IsInfinity.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library


Public Module Extensions_201

	''' <summary>
	'''     Returns a value indicating whether the specified number evaluates to negative or positive infinity.
	''' </summary>
	''' <param name="d">A double-precision floating-point number.</param>
	''' <returns>true if  evaluates to  or ; otherwise, false.</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function IsInfinity(d As [Double]) As [Boolean]
		Return [Double].IsInfinity(d)
	End Function
End Module


